actors/runtime/src/actor_error.rs

```diff
@@ -1,6 +1,41 @@
 use fvm_shared::error::ExitCode;
 use thiserror::Error;
 
+mod abort {
+    use crate::ActorError;
+    use fvm_shared::error::ExitCode;
+
+    #[derive(thiserror::Error, Debug)]
+    #[error("abort error")]
+    pub enum Abort {}
+
+    #[cfg(feature = "fil-actor")]
+    fn maybe_abort(exit_code: ExitCode, msg: Option<&str>) -> ! {
+        fvm_sdk::vm::abort(exit_code as u32, msg);
+    }
+    #[cfg(not(feature = "fil-actor"))]
+    fn maybe_abort(exit_code: ExitCode, msg: Option<&str>) -> ! {
+        // TODO: maybe not panic, needs discussion what we want here
+        panic!("Abort: {}: {:?}", exit_code, msg);
```

Comment:
It works well in Go. If we can reliably catch the panic, e.g. in test harnesses, and inspect the exit code, then this could be good. My read of the Rust docs suggested this was even more frowned upon than in Go, though. cc @ZenGround0 

I would make the `msg` non-optional.